sales_team_invoiced_target_domain/model/res_company.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    sales_team_invoiced_domain = fields.Char(
+        default="['|', '|', "
+        "['payment_state', '=', 'in_payment'], "
+        "['payment_state', '=', 'paid'], "
+        "['payment_state', '=', 'reversed']]"
```

Comment:
I think this can be improved using the `in` operator.

```suggestion
        default="[('payment_state', 'in', ['in_payment', 'paid', 'partial', 'reversed'])]"
```

![image](https://github.com/user-attachments/assets/029dca03-2110-4444-b580-97a2e1ad4c3d)

Why didn't you include the `partial` option?
![image](https://github.com/user-attachments/assets/4bbc29d6-0336-4fff-a2f9-edaa9d9f5b37)

Reply:
I’m not adding it because that’s how it’s done in the core. https://github.com/odoo/odoo/blob/18.0/addons/sale/models/crm_team.py#L70